rabbitmq/datadog_checks/rabbitmq/rabbitmq.py

```diff
@@ -158,6 +158,8 @@ def _get_config(self, instance):
             raise Exception('Missing "rabbitmq_api_url" in RabbitMQ config.')
         if not base_url.endswith('/'):
             base_url += '/'
+
+        disable_nodes = instance.get('disable_node_metrics')
```

Comment:
missing default value `instance.get(<name>, <default_value>)` otherwise it gets set to `None` which may have unexpected consequences.